### ISSUE TITLE
How can I create a df using your library - fiscal445?

### ISSUE THREAD

**Issue by adioleope** [dv8edroute/fiscal445#7]
Could you please help me understand how I could generate 4-4-5 calendar dataframe from dates (let's say 01/01/21 to 03/03/21) in a sense that it displays the week number, day and the quarter it belongs to?

For instance, 
```
  date  |   day   | week_no | yyyy-mm | quarter
01/01/21| Friday  |    40   | 2021-01 | Q4 2021
02/01/21| Saturday|    40   | 2021-01 | Q4 2021
.
.
24/01/21| Sunday  |    44   | 2021-02 | Q4 2021
.
.
01/04/21| Thursday|    53   | 2021-03 | Q4 2021
.
04/04/21| Sunday  |    1    | 2021-04 | Q1 2022

```

<img width="1067" alt="Screenshot 2021-04-10 at 5 57 31 PM" src="https://user-images.githubusercontent.com/29918606/114265952-3abf9180-9a26-11eb-8592-eaeb76ec4fcc.png">

**Comment by adioleope** [comment]
I wrote this `fc5.cal = fc5.Calendar('2020-04-01','sat').build()` and it gives the correct result till January-21. I am attaching the df where it goes wrong..

`

January | 1 | 2021-01-30 00:00:00
-- | -- | --
January | 2 | 2021-02-06 00:00:00
January | 3 | 2021-02-13 00:00:00
January | 4 | 2021-02-20 00:00:00
February | 1 | 2021-02-27 00:00:00
February | 2 | 2021-03-06 00:00:00
February | 3 | 2021-03-13 00:00:00
February | 4 | 2021-03-20 00:00:00
February | 5 | 2021-03-27 00:00:00
March | 1 | 2021-04-03 00:00:00

`
January repeats itself again for some reason. How to avoid that?


Basically, how may I use your library to generate all corresponding week number, week day, yyyy-mm, quarter according to the screenshot attached for all dates between a given date range? Would be grateful if you could help me out here.
